pkg/features/operator_features.go

```diff
@@ -68,6 +68,11 @@ const (
 	// beta: v1.0
 	// RestartCluster uses crdb binary to generate self signed certifcates
 	ClusterRestart featuregate.Feature = "ClusterRestart"
+	// owner: @alina
+	// alpha: v0.1
+	// beta: v1.0
+	// PVCPruner uses crdb binary to prune PVC on decommission
+	PVCPruner featuregate.Feature = "PVCPruner"
```

Comment:
Can we change this to `AutoPrunePVC` or something so the name is a bit more descriptive?